### ISSUE TITLE
Testing coverage?

### ISSUE THREAD

**Issue by Sathors** [thinca/vim-themis#38]
Do you think it would be possible to implement testing coverage?
Or would you know of another tool that fulfill this purpose?

Thanks for the great work!

**Comment by thinca** [comment]
I'm planning the coverage feature, but this is very difficult.
Please don't get your hopes up. 😅

**Comment by Sathors** [comment]
@thinca, indeed, it would be really nice, even though I have no idea how you could achieve that ^^.

**Comment by thinca** [comment]
Now you can use [covimerage](https://github.com/Vimjas/covimerage).
Here is [example to introduce](https://github.com/vim-jp/vital.vim/pull/554/files).
